_posts/2025-07-27-5.13.0-released.md

```diff
@@ -0,0 +1,74 @@
+
+You can download Luanti from [our website](https://www.luanti.org/downloads/), check out the code on [GitHub](https://github.com/luanti-org/luanti/), or read the [full changelog](https://docs.luanti.org/about/changelog/#5120--5130).
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Persistent Object IDs](#persistent-object-ids)
+- [Working with Nodes](#working-with-nodes)
+- [Show Them Their Inventory](#show-them-their-inventory)
+- [A Couple of Controls](#a-couple-of-controls)
+- [Bugs-B-Gone](#bugs-b-gone)
+- [What's Next?](#whats-next)
+
+## Persistent Object IDs
+
+Let's start with the big one: Object IDs! Why is this important? You see, before persistent object IDs, developers had to track unloaded entities and objects manually using staticdata or other gross tricks. With [#14135](https://github.com/luanti-org/luanti/pull/14135), the engine provides `Object:get_guid()`, returning a globally unique identifier that will remain consistent across unloads and world sessions until the entity is destroyed. Never loose track of your objects ever again!
+
+## Working with Nodes
+
+Here's a handful of methods you can use when working with nodes and chunks.
+
+* `core.get_node_raw()` to query nodes from the map, but faster, and with a content ID ([#16265](https://github.com/luanti-org/luanti/pull/16265))
+* `VoxelManip:initialize()` to create an empty VM storage object ([aa1bab2](https://github.com/luanti-org/luanti/commit/aa1bab21564cdac01ed2eaed6c2284fc1497626a))
+* `core.get_mapgen_chunksize()` to determine the configured chunksize for your custom mapgens ([#16289](https://github.com/luanti-org/luanti/pull/16289))
+
+Handy, no?
+
+## Show Them Their Inventory
+
+It might not seem significant, but manually pulling up a player's inventory has been a long-requested feature. Now, by passing an empty string (`""`, reserved for the player inventory) as the formname to `core.show_formspec()`, you can open the player's inventory without their input ([#15963](https://github.com/luanti-org/luanti/pull/15963)). I bet this could be used in some creative ways...
+
+## A Couple of Controls
+
+Firstly, you can now cycle through inventory tabs using `ctrl+tab` (`+shift` to cycle in reverse). This is a standard keybind for application tab switching, so we are happy to see it arrive in Luanti ([#16167](https://github.com/luanti-org/luanti/pull/16167)).
+
+Secondly, the keybind modifier for exiting to the main menu is configurable ([#16250](https://github.com/luanti-org/luanti/pull/16250)). `ESC` plus whatever you desire. More importantly, it is no longer set by default, preventing players from accidentally leaving the game due to pressing `Shift+ESC` during quick movements. Now you can play CTF in (relative) peace.
+
+## Bugs-B-Gone
+
+The poor insects are being squashed left and right! You may be happy to learn that a plethora of issues have been resolved regarding:
```

Comment:
The idea of squashing insects like a wack-a-mole game but with juice attached is not the best image I'd like to picture in my head :c